pom.xml

```diff
@@ -112,7 +112,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.19.1</version>
                 <configuration>
-                    <argLine>-Xmx100m -Xms100m</argLine>
+                    <argLine>-Xmx130m -Xms130m</argLine>
```

Comment:
Had to increase this because of the new test that builds multiple LM/CH profiles. Obviously it would be better to run these tests on a smaller map, but then we would need to add another map and some gpx data and in core we are using the same limit :)

Reply:
:+1: